app/src/main/java/waiter/Listener.java

```diff
@@ -0,0 +1,15 @@
+package waiter;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+
+public class Listener implements Awaiter {
+    ServerSocket server;
+
+    Listener(ServerSocket server) {
+        this.server = server;
+    }
+    public Connection awaitClient() throws IOException {
```

Comment:
```suggestion
    }
    
    public Connection awaitClient() throws IOException {
```

Reply:
Resolved in [this commit](https://github.com/jonmanahan/waiter/pull/2/commits/e53fdbc1eec22480eae57e79838d0f1186c16f90), if it resolves the issues please hit resolve conversation if possible, thank you all a ton!